File: Java(Udemy)/Adivinha/app/src/main/java/adivinha/nivaldojr/com/adivinha/MainActivity.java
package adivinha.nivaldojr.com.adivinha;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    private Button botaoJogar;
    private TextView textoResultado;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        botaoJogar = (Button) findViewById(R.id.botaoJogarId);
        textoResultado = (TextView) findViewById(R.id.resultadoId);

        botaoJogar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Random randomico = new Random();

                int num = randomico.nextInt(11);
                textoResultado.setText("Número escolhido: " + num);
            }
        });
    }
}

File: Kotlin(Unifavip Wyden)/Login/app/src/main/java/aula3/nivaldojr/com/login/MainActivity.kt
package aula3.nivaldojr.com.login

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val editTextName = findViewById<EditText>(R.id.name)
        val editTextUsername = findViewById<EditText>(R.id.username)
        val editTextPassword = findViewById<EditText>(R.id.password)

        val buttonCancel = findViewById<Button>(R.id.buttonCancel)
        val buttonLogin = findViewById<Button>( R.id.buttonLogin)

        buttonLogin.setOnClickListener(){
            if(editTextUsername.text.toString()=="user123"
                    && editTextPassword.text.toString() == "abc123") {
                var name = editTextName.text.toString()

                val intent = Intent(this, WelcomeActivity::class.java)

                intent.putExtra("name_welcome", name)

                startActivity(intent)

            }else if(editTextUsername.text.toString()!="user123"){
                Toast.makeText(this, "Invalid user!", Toast.LENGTH_SHORT).show()
            }else if(editTextPassword.text.toString() != "abc123"){
                Toast.makeText(this, "Invalid password!", Toast.LENGTH_SHORT).show()
            }
        }

        buttonCancel.setOnClickListener(){
            Toast.makeText(this, "Bye :)", Toast.LENGTH_SHORT).show()
            finish()
        }
    }
}
